benchmark/scripts/Benchmark_Driver

```diff
@@ -129,6 +129,12 @@ def get_current_git_branch(git_repo_path):
         ['git', '-C', git_repo_path, 'rev-parse',
          '--abbrev-ref', 'HEAD'], stderr=subprocess.STDOUT).strip()
 
+def get_git_head_ID(git_repo_path):
+    """Return the short identifier for the HEAD commit of the repo
+        `git_repo_path`"""
+    return subprocess.check_output(
+        ['git', '-C', git_repo_path, 'rev-parse',
+         '--short', 'HEAD'], stderr=subprocess.STDOUT).strip()
```

Comment:
Can you use the full hash here? The reason why is that over time as history grows, a prefix collision becomes much more likely to happen. I have noticed this happen a few times with swift recently. It would be unfortunate if this was attached to a bug report and we could not tell the exact revision that it came from. That being said, I am not that wedded to this if you think I am being too paranoid.